Description: Disable binary-compatibility plugin
 The plugin requires `javaparser` and `japicmp` which are not in Debian yet
Author: Kai-Chung Yan
Forwarded: not-needed
--- a/buildSrc/build.gradle
+++ b/buildSrc/build.gradle
@@ -37,6 +37,8 @@
     exclude 'org/gradle/plugins/pegdown/**'
 }
 
+sourceSets.main.groovy { exclude 'org/gradle/binarycompatibility/**' }
+
 dependencies {
     compile 'org.ow2.asm:asm-all:6.2.1@jar'
     compile gradleApi()
@@ -46,9 +48,6 @@
     testCompile 'junit:junit:4.12@jar'
     testCompile 'org.spockframework:spock-core:1.0-groovy-2.4@jar', 'cglib:cglib-nodep:3.2.5', 'org.objenesis:objenesis:1.2'
     testCompile 'org.hamcrest:hamcrest-core:1.3'
-
-    compile 'me.champeau.gradle:japicmp-gradle-plugin:0.2.4'
-    compile 'com.github.javaparser:javaparser-core:2.4.0'
 }
 
 ext.isCiServer = System.getenv().containsKey("CI")
--- a/subprojects/distributions/distributions.gradle
+++ b/subprojects/distributions/distributions.gradle
@@ -131,5 +131,3 @@
     systemProperty 'org.gradle.public.api.includes', publicApiIncludes.join(':')
     systemProperty 'org.gradle.public.api.excludes', publicApiExcludes.join(':')
 }
-
-apply from: 'binary-compatibility.gradle'
